prediction_market_agent/agents/known_outcome_agent/benchmark.py

```diff
@@ -1,3 +1,4 @@
+# type: ignore
```

Comment:
What's this for?

Reply:
Mypy complains about tons of type errors in this file, which isn't part of this PR

Reply:
Well that's weird, but it works just fine in main branch, so we shouldn't ignore it:

![Screenshot by Dropbox Capture](https://github.com/gnosis/prediction-market-agent/assets/15619339/5c3b1205-3014-4be2-b271-e9907ae3126b)

Reply:
My mistake, mypy working fine now

Reply:
In that case this can be removed?

```suggestion
```